homeassistant/components/solarlog/sensor.py

```diff
@@ -96,6 +101,16 @@ def state(self):
         """Return the state of the sensor."""
         return self._state
 
+    @property
+    def state_class(self):
+        """Return the state class of the sensor."""
+        return self._state_class
+
+    @property
+    def device_class(self):
+        """Return the state class of the sensor."""
+        return self._device_class
+
```

Comment:
with the entity description, you can remove these methods.

```suggestion
```

Reply:
removed